cmd/arc/services/k8s_watcher.go

```diff
@@ -2,11 +2,11 @@ package cmd
 
 import (
 	"fmt"
+	k8swatcher "github.com/bitcoin-sv/arc/internal/k8s_watcher"
```

Comment:
No aliases needed here, as this rule has been excluded

Reply:
Removed